plugins/inputs/mysql/mysql.go

```diff
@@ -820,98 +645,36 @@ func (m *Mysql) gatherBinaryLogs(db *sql.DB, serv string, acc telegraf.Accumulat
 // the mappings of actual names and names of each status to be exported
 // to output is provided on mappings variable
 func (m *Mysql) gatherGlobalStatuses(db *sql.DB, serv string, acc telegraf.Accumulator) error {
-	// If user forgot the '/', add it
-	if strings.HasSuffix(serv, ")") {
-		serv = serv + "/"
-	} else if serv == "localhost" {
-		serv = ""
-	}
-
```

Comment:
Why is this removed?

Reply:
The user can't forget the slash at the end, the driver will bubble up an error saying they forgot the slash at the end and the plugin will fail. It was inconsistent with the rest of the plugin and the default behavior instructs the user on how to fix it rather than continuing incorrectly.